packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2944,6 +2945,44 @@ export default class Meeting extends StatelessWebexPlugin {
     return reject('Meeting:index#sendDTMF --> cannot send DTMF, meeting does not have a connection to the "locus" call control service. Have you joined?');
   }
 
+  /**
+   * Sends request to change layout type for the current meeting for the specific participant/device only
+   * @param {String} layoutType a layout type that should be available in meeting constants LAYOUT_TYPES
+   * @returns {Promise}
+   * @public
+   * @memberof Meeting
+   */
+  changeVideoLayout(layoutType) {
+    const reject = (message) => {
+      LoggerProxy.logger.error(message);
+
+      return Promise.reject(new Error(message));
+    };
+
+    // TODO: We need a real time value for Audio, Video and Share send indicator
+    if (this.mediaProperties.mediaDirection.receiveVideo === true && this.mediaProperties.remoteStream) {
+      if (LAYOUT_TYPES.includes(layoutType)) {
+        return this.meetingRequest
+          .changeVideoLayout({
+            locusUrl: this.locusInfo.self.url,
+            deviceUrl: this.deviceUrl,
+            layoutType
+          })
+          .then((response) => {
+            if (response && response.body && response.body.locus) {
+              this.locusInfo.onFullLocus(response.body.locus);
+            }
+
+            return Promise.resolve();
```

Comment:
This call isn't necessary. You're already in a Promise chain

Reply:
Addressed!